README.md

```diff
@@ -511,11 +504,44 @@ In order to use environment variable in deployment or cronjob, you will have to
    you can either provide `nameSuffix` which means name added after prefix ```<applicationName>-``` or static name with ```name``` of secret
 
    **Note:** first key after ``envFrom`` is just used to uniquely identify different objects in ``envFrom`` block. Make sure to keep it unique and relevant 
+
+
+### Configuring probes
+
+By default probe handler type is `httpGet`. You just need to override `port` and `path` as per your need.
+
+```
+  livenessProbe:
```

Comment:
@bilalbokharee don't i have to first set enabled true?

Reply:
This is an example of httpGet config only. Should I add enabled keyword here as well? This snippet does not indicate enabling.

Reply:
shouldn't examples be set based on our chart? if I am reading this example I will miss it easily and just copy and paste and then struggle